clients/azuredevopsrepo/search_e2e_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azuredevopsrepo
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/ossf/scorecard/v5/clients"
+)
+
+var _ = Describe("E2E TEST: azuredevopsrepo.searchHandler", func() {
+	Context("Search - Azure DevOps", func() {
+		It("Should return search results", func() {
+			Skip("We don't have a public Azure DevOps repo to test against")
```

Comment:
optional: you can remove this `Skip()` since you have it gated on `TEST_AZURE_DEVOPS_EXTERNAL` in the suite.

Otherwise `make e2e-azure-devops` skips it for you too